Sources/jab/Identifiable.swift

```diff
@@ -10,11 +10,5 @@ import Foundation
 public protocol JSONAPIIdentifiable {
     var identifier: String { get set }
     
-    static var codingKey: String { get }
-}
-
-extension JSONAPIIdentifiable {
-    static var codingKey: String {
-        return "id"
-    }
+    static var jsonTypeIdentifier: String { get }
```

Comment:
what is the difference between jsonType and plain identifier?

Reply:
oba su dio json:api specifikacije - https://jsonapi.org/format/#document-resource-object-identification

identifier je, uobicajeno, neki UUID string
jsonTypeIdentifier je neki string koji uniquely identifiea tip objekta